api2/pkg/webapp/cases.go

```diff
@@ -87,6 +90,15 @@ func (h *Handler) Case(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
+	if id != "new" {
+		teamRes, err := h.teamClient.Get(ctx, kase.TeamID)
+		if err != nil {
+			http.Error(w, err.Error(), http.StatusInternalServerError)
```

Comment:
@senyaoh @ludydoo this implies that if we have no case team selected, we will get an error page. Is this the correct behaviour?

Reply:
In that case, I believe the method would return `(nil, nil)`, so no error, normally